<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register_info</class>
 <widget class="QWidget" name="Register_info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{font: 75 14pt &quot;Arial&quot;;
background-color: rgb(49, 52, 59);
color: rgb(255, 255, 255);}

</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>50</y>
     <width>681</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-top:1.5px solid rgb(64,64,64);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>用户注册信息管理</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>501</width>
     <height>540</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>年龄</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>民族</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>性别 </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>左右利手</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>工作经历</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>教育水平</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>联系电话</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>联系地址</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>身体健康状况</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>体育锻炼类型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>常玩游戏(类型,名称)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>日常娱乐方式</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEdit_name"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_age"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_nation"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_sex">
        <property name="placeholderText">
         <string>‘男’或‘女’</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_handy">
        <property name="placeholderText">
         <string>‘左’或‘右’</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_work"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_education"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_phone"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_address"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_health"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_sport"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_game"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_entertainment"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_saveInfo">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>620</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton#pushButton_register{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0  rgb(6, 191, 255), stop:1 rgb(44, 114, 255) );
	color: rgb(255, 255, 255);
	border-radius:3px;		
	font: 12pt &quot;Adobe 黑体 Std R&quot;;
}
QPushButton#pushButton_register:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0  rgb(20, 204, 255), stop:1 rgb(11, 141, 255) );
	border-style:none;
	border-radius:3px;
	font: 12pt &quot;Adobe 黑体 Std R&quot;;
}

QPushButton#pushButton_register:pressed{
	
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0  rgb(20, 204, 255), stop:0.5 rgb(5, 191, 255), stop:1 rgb(11, 141, 255) );
	border-style:none;
	border-radius:3px;	
	font: 12pt &quot;Adobe 黑体 Std R&quot;;
}
</string>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
